code/modules/projectiles/guns/ballistic.dm

```diff
@@ -92,7 +99,7 @@
 	/// Check if you are able to see if a weapon has a bullet loaded in or not.
 	var/hidden_chambered = FALSE
 
-	///Gun internal magazine modification and misfiring
+	/// Gun internal magazine modification and misfiring ///
```

Comment:
This is describing a bunch of variables, so just normal comment this with //

Reply:
Got it.